src/main/java/com/google/sps/data/Event.java

```diff
@@ -48,119 +46,54 @@
-      this.strEndTime = createStrTime(this.date, this.endTime);
-    } catch (Exception e) {
-      e.printStackTrace();
-    }
+  public Event(String name, String address, LocalDateTime startTime, 
+              int travelTime) {
+    this(name, address, startTime, travelTime, HOUR);
   }
 
   /**
-   * function that builds string w correct format for calendar-script.js
+   * checks that user inputs travelTime is valid
    * 
-   * @param date date from user input travel day. Cannot be null.
-   * @param time time to to start or end event. Must be within bounds
+   * @param time amount of time spent traveling (minutes)
    */
-  private static String createStrTime(String date, int time) throws Exception {
-    if (date == null) {
-      throw new NullPointerException("Date cannot be null");
-    }
-
+  private static void checkTravelTime(long time) {
     if (time < MIN_POSSIBLE_TIME) {
       throw new IllegalArgumentException("Time cannot be less than " + MIN_POSSIBLE_TIME);
     }
 
     if (time >= MINUTES_IN_A_DAY) {
-      throw new IllegalArgumentException("Time cannot be more than " + MINUTES_IN_A_DAY);
-    }
-
-    String output = "";
-    output += date + "T" + Integer.toString(time).substring(0, 2) + ":" + 
-      Integer.toString(time).substring(2, 4) + ":00";
-
-    try {
-      SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
-      Date d = sdf.parse(output);
-    } catch (Exception e) {
-      e.printStackTrace();
+      throw new IllegalArgumentException("Time cannot be more than or equal to" + MINUTES_IN_A_DAY);
     }
-    
-    return output;
-  }
-
-  /**
-   * function that calculates endTime given start and timeSpent
-   * 
-   * @param timeAtLocation time that the user spends at a POI
-   */
-  private int calculateEndTime(int timeAtLocation) {
-    int minsStartTime = (this.startTime / 100) * 60 + (this.startTime % 100);
-
-    return Integer.parseInt(convertToFormat(timeAtLocation + minsStartTime));
   }
 
-  /**
-   * function that converts minutes into hhmm format
-   * 
-   * @param time time in minutes
-   */
-  public static String convertToFormat(int time) {
-    if (time < MIN_POSSIBLE_TIME) {
-      throw new IllegalArgumentException("Time cannot be less than " + MIN_POSSIBLE_TIME);
-    }
-
-    if (time >= MINUTES_IN_A_DAY) {
-      throw new IllegalArgumentException("Time cannot be more than " + MINUTES_IN_A_DAY);
-    }
-
-    String numHours = Integer.toString(time / 60);
-    String numMins = Integer.toString(time % 60);
-
-    // add 0's to front of number if neccessary
-    if (Integer.parseInt(numHours) < 10) {
-      numHours = "0" + numHours;
-    } 
-
-    if (Integer.parseInt(numMins) < 10) {
-      numMins = "0" + numMins;
-    }
-
-    return numHours + numMins;
+  public static String getProperDateFormat(LocalDateTime ldt) {
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss");
```

Comment:
It looks like you're using the ISO_LOCAL_DATE_TIME format, if that's the case, you can just write:

"return DateTimeFormatter.ISO_LOCAL_DATE_TIME.format(ldt);"

And if this is the case, then I'm not sure how necessary this method actually is. It's referring to a 'proper' date format, but what does that actually mean? A user of your event class could want the time in a multitude of formats.